smarts/core/tire_models.py

```diff
@@ -116,6 +116,10 @@ def _calculate_slip_angles(chassis, client, action):
 
         return slip_angles
 
+    def apply_tire_forces(self, chassis, client, action):
+        """Applies tire forces"""
+        raise NotImplementedError
```

Comment:
Will this throw an error since it is being used in `smarts/core/chassis.py` but not implemented?